src/pages/index.tsx

```diff
@@ -104,7 +106,7 @@ export const Home = () => {
                   fontFamily='Lab Grotesque'
                   pb={2}
                   textAlign='left'
-                  variant='subtitle2'
+                  variant='p'
```

Comment:
<img width="663" alt="Screenshot 2023-02-17 at 1 40 39 PM" src="https://user-images.githubusercontent.com/56411016/219799105-1a5de8ad-7ec4-423d-a969-2d6ccc7ed13f.png">

Reply:
Does `variant='p'` fix this error message? `Typography` by default has a variant of `p`. I would think this should be `subtitle1` or `subtitle2`.

Reply:
Agreed about it being `subtitle1` or `subtitle2`. `variant='p'` did fix the error message though and didn't change the UI.